Iterator-View.md

```diff
@@ -243,8 +245,8 @@ is the same as
 ### Signatures
 
 ```cpp=
-  Domain cartesian(int i0, int i1, int j0, int j1, int k0, int k1);
-  Domain unstructured(Location, int hor0, int hor1, int k0, int k1);
+  NamedRange named_range(Axis, int begin, int end)
+  Domain domain(NamedRange...);
```

Comment:
There is no need to distinguish Cartesian and unstructured by its domain as anyway we need more information at compile/generation time about the concrete offset interpretation. E.g. max neighbors for a neighbor table.